grails-app/domain/org/pih/warehouse/shipping/Shipment.groovy

```diff
@@ -415,6 +415,9 @@ class Shipment implements Comparable, Serializable {
     User getShippedBy() {
         Event shippedEvent = events?.find { it.eventType?.eventCode == EventCode.SHIPPED }
         if (shippedEvent) {
+            if (requisition?.electronicType) {
```

Comment:
can you add a comment here explaining why we want to do this just for e-requisitions? From the description of the ticket, this is just a more accurate default value in that scenario